Packages/vcs/Lib/vcsvtk/fillareautils.py

```diff
@@ -33,8 +34,14 @@ def make_patterned_polydata(inputContours, fillareastyle=None,
     # Scaled the size to 2 times to make the pattern image of a finer resolution
     xBounds = bounds[1] - bounds[0]
     yBounds = bounds[3] - bounds[2]
-    xres = int(4.0*xBounds)
-    yres = int(4.0*yBounds)
+    if xBounds <= 1 and yBounds <= 1 and size is not None:
+        xBounds *= size[0]
```

Comment:
@doutriaux1 It is related to render window size currently, yes.

@sankhesh Ah, I think I understand our problem (and the solution). `fillarea`s are being set up with normalized viewport coordinates, not viewport coordinates. I'm going to see what I can do about fixing that issue; then the code here should work properly (as the bounds will be greater than 1).